src/sentry/replays/usecases/ingest/dom_index.py

```diff
@@ -0,0 +1,145 @@
+import time
+import uuid
+import zlib
+from typing import Iterator, List, Literal, Optional, TypedDict
+
+from django.conf import settings
+
+from sentry.utils import json, kafka_config
+from sentry.utils.pubsub import KafkaPublisher
+
+replay_publisher: Optional[KafkaPublisher] = None
+
+ReplayActionsEventPayloadClick = TypedDict(
+    "ReplayActionsEventPayloadClick",
+    {
+        "node_id": int,
```

Comment:
is `node_id` something people would ever search on? it's internal to RRWeb right?

Reply:
Its not something we would want to search for but definitely useful data that could be returned to the client.

Reply:
ah good point. makes sense.